core-build/gulp-core-build/src/tasks/JestTask.ts

```diff
@@ -1,11 +1,15 @@
 // Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
 // See LICENSE in the project root for license information.
 import * as path from 'path';
-import { GulpTask} from './GulpTask';
+import { GulpTask } from './GulpTask';
 import { IBuildConfig } from '../IBuildConfig';
 import * as Gulp from 'gulp';
-import * as Jest from 'jest-cli';
 import * as glob from 'glob';
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const { runCLI } = require('@jest/core');
+// import { runCLI } from 'jest/core';
```

Comment:
@sachinjoseph Is this leftover from work in progress?

Reply:
Yes. Will fix.